packages/bot-skeleton/src/scratch/blocks/Binary/Trade Definition/trade_definition.js

```diff
@@ -99,6 +102,13 @@ Blockly.Blocks.trade_definition = {
         };
     },
     onchange(event) {
+        if (event.type === 'ui' && !this.isInit) {
+            this.isInit = true;
+            initErrorHandlingListener('keydown');
```

Comment:
The default value is `keydown` so we don't need to pass it from here. let's either remove the default value or make this parameter optional @rupato-deriv

Reply:
done ✅